### ISSUE TITLE
Few useful working examples !

### ISSUE THREAD

**Issue by jonshouse1** [themadinventor/esptool#38]
The notes espressif have released for flashing the AT command interpreter onto the module seem incomplete to the point of useless.

I have spent 6 hours now trying to get the latest ESP release AT command interpreter onto the module with no sucess.  Esptool itself seems to work but the result of my tries so far is just a solid red and blue LED.

Can somebody help me generate a wiki entry on how to flash the latest AT command interpreter onto modules.
http://bbs.espressif.com/viewtopic.php?f=5&t=346

# pwd
....... 1.0.1_may2015/esp_iot_sdk_v1.0.1/bin

# esptool.py --port /dev/ttyUSB0  write_flash 0x00000 boot_v1.2.bin 0x01000 at/user1.1024.new.bin  0x7c000 esp_init_data_default.bin 0x7e000 blank.bin

So far I have tried lots of variations around this kind of thing, none seem to produce a working module. I get the "erasing flash" and "writing" messages correctly.

In short how do I use esptool.py to get the AT command interpreter onto a module (from any state, IE flashing all regions with code or defaults so that the flash process will always work!)

http://bbs.espressif.com/download/file.php?id=293

Thanks,
Jon

**Comment by projectgus** [comment]
Hi Jon,

I think you probably got bit by the long-running "SPIEraseFlash" bug (in ESP ROM), that only just got worked around in esptool.py #53. The command line you wrote looks otherwise good.

I'm going to close the issue, but if you try this again with the latest esptool.py and it still breaks then please reopen.

Angus
